### DISCUSSION TITLE
First presentation: show off your Big Ideas

### DISCUSSION THREAD

**Discussion by aumouvantsillage** [sozi-projects/Sozi-guide#2] [The Sozi Guide in English]
This chapter introduces the basic features of Sozi for beginners.
It is a translation of the French version.

[Follow this link to read it](http://sozi.guide/en/first-presentation.html)

Please share your impressions about this chapter. Here are a few questions that we can address here, but other topics are welcome:

* How can we improve the English translation?
* Are the instructions easy to understand and the explanations clear enough?
* Do you think that some important topics have been omitted from this chapter?
* Do you think that this chapter contains too much information? Which sections do you think could be moved to another chapter?

**Comment by ecodiv** [comment]
I would say that this chapter is pretty complete and things are well explained. Compliments,  and a big thanks for the software and explanations!
